keps/sig-network/2059-servicecidrblock-migration/kep.yaml

```diff
@@ -0,0 +1,51 @@
+title: Service CIDR Block Migration
+kep-number: NNNN
+authors:
+  - "@uthark"
+owning-sig: sig-network
+participating-sigs:
+  - sig-aaa
+  - sig-bbb
+status: provisional
+creation-date: 2020-10-01
+reviewers:
+  - TBD
+  - "@alice.doe"
+approvers:
+  - TBD
+  - "@oscar.doe"
+prr-approvers:
+  - TBD
+  - "@bob.doe"
+see-also:
+  - "/keps/sig-aaa/1234-we-heard-you-like-keps"
+  - "/keps/sig-bbb/2345-everyone-gets-a-kep"
+replaces:
+  - "/keps/sig-ccc/3456-replaced-kep"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.19"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.19"
```

Comment:
a little confused did this already go alpha? but there wasn't a KEP for it in the last release?

Reply:
this comes from `kep.yaml` template, didn't touch it.

Reply:
the kep.yaml template is an example so you'll need to update it so that it's applicable to your kep =)